### PR TITLE
Add h1 and h2 to the components list

### PR DESCRIPTION
**Pull request by jpa198**
Add function to return the currently available basic components
Refactor read_ascii_folder to load arbitrary basic components, including h1, h2
Formatting

### CODE REVIEW

**Comment by jpa198** [comment]
I have verified that that the script runs and produces IMs for H1, H2. Any ideas on validation of the results?

**Comment by jasonmotha** [comment]
Verification can be geom(000, 090) ~= geom(h1, h2) ?

**Review by jasonmotha — APPROVED** [review]
I refrain from making any comments as they fall back to the original author of these functions... Happy as long as validation makes sense.

**Comment by jpa198** [comment]
> Verification can be geom(000, 090) ~= geom(h1, h2) ?

That didn't turn out as close as I'd hoped...

File: IM_calculation/IM/read_waveform.py [hunk 1]
```diff
@@ -168,8 +168,9 @@ def read_ascii_folder(path, station_names, comp, units="g"):
                 c: open(os.path.join(path, f"{station}.{c.str_value}")) for c in comp
             }
         except IOError:
+            files_names = f"{station}.{{{[c.str_value for c in comp]}}}"
```
Comment (claudio525) [inline IM_calculation/IM/read_waveform.py:171]: Nitpicking here, but given that this variable is only used once. I suggest to either:
a) not define the variable and put the logic in the string formatting OR
b) define a variable, but put all the logic in its definition (i.e. currently the join is done in the string, and the rest is done in the variable definition), e.g. something like
```python
file_paths = os.path.join(path, f"{station}.{{{[c.str_value for c in comp]}}})
```

Comment (jpa198) [reply IM_calculation/IM/read_waveform.py:171]: Ah yeah, option b makes sense.
Option a includes more nesting " marks than I want to have to think about.

**Comment by jpa198** [comment]
> > Verification can be geom(000, 090) ~= geom(h1, h2) ?
> 
> That didn't turn out as close as I'd hoped...

After discussion with Robin, and manual investigation, the H1, H2 outputs look alright. This PR is really only about loading them, and this seems to have worked

**Review by jasonmotha — APPROVED** [review]

**Review by shot0829 — APPROVED** [review]

**Review by claudio525 — APPROVED** [review]

**Pull request merged**